widget/tests/test_Toaster.html

```diff
@@ -7,60 +6,66 @@
-		dojo.require("dojo.parser");	// scan page for widgets and instantiate them
+	<script src="../../../dojo/dojo.js" data-dojo-config="isDebug:true, async: 1"></script>
+	<script>
+		require([
+			"dojo/parser",
+			"dojo/topic",
+			"dojo/on",
+			"dojo/dom",
+			"dijit/registry",
+			"dijit/Tooltip",
+			"dojox/widget/Toaster",
+			"dojo/domReady!"
+		], function (parser, topic, on, dom, registry) {
+			parser.parse();
+			var toast = registry.byId("toast");
+			function showTestMessage(){
+				topic.publish("testMessageTopic",
+					"This is a message! It's kind of long to show message wrapping."
+				);
+			}
+			function showAnotherMessage(){
+				topic.publish("testMessageTopic",
+					{
+						message: "This is another message!",
+						type: "warning",
+						duration: 500
+					}
+				);
+			}
+			function showYetAnotherMessage(){
+				topic.publish("testMessageTopic",
+					{ message: "This is yet another message!" }
+				);
+			}
 
-		dojo.require("dijit.Tooltip");
-		
-		var toast = null;
-		function showTestMessage(){
-			dojo.publish("testMessageTopic", 
-				[ "This is a message! It's kind of long to show message wrapping."]
-			);
-		}
-		function showAnotherMessage(){
-			dojo.publish("testMessageTopic", 
-				[{
-					message: "This is another message!", 
-					type: "warning", 
-					duration: 500
-				}]
-			);
-		}
-		function showYetAnotherMessage(){
-			dojo.publish("testMessageTopic", 
-				[{ message: "This is yet another message!" }]
-			);
-		}
-
-		dojo.addOnLoad(function(){
-			toast = dijit.byId("toast");
+			on(dom.byId("submitButton1"), "click", showTestMessage);
+			on(dom.byId("submitButton2"), "click", showAnotherMessage);
+			on(dom.byId("submitButton3"), "click", showYetAnotherMessage);
 		});
+
 	</script>
 </head>
 <body class="tundra">
-	<div dojoType="dojox.widget.Toaster" id="toast" 
-		positionDirection="br-left" duration="0" 
-		messageTopic="testMessageTopic"></div>
+	<div data-dojo-type="dojox/widget/Toaster" id="toast"
+		data-dojo-props="positionDirection: 'br-left', duration:0, messageTopic:'testMessageTopic'"></div>
 
-	<div dojoType="dojox.widget.Toaster" id="toast2" 
-		separator="&lt;hr&gt;" positionDirection="bl-up" 
-		messageTopic="testMessageTopic"></div>
+	<div data-dojo-type="dojox/widget/Toaster" id="toast2"
+		data-dojo-props="separator: '&lt;hr&gt;', positionDirection: 'bl-up', messageTopic:'testMessageTopic'"></div>
 
-	<button type="submit" 
-		onclick="showTestMessage();">Click to show message</button>
-	<button type="submit" 
-		onclick="showAnotherMessage();">Click to show another message</button>
-	<button type="submit" 
-		onclick="showYetAnotherMessage();">Click to show yet another message</button>
+	<button type="submit" id="submitButton1"
+		>Click to show message</button>
+	<button type="submit" id="submitButton2"
+		>Click to show another message</button>
+	<button type="submit" id="submitButton3"
+		>Click to show yet another message</button>
```

Comment:
Not sure why you changed the click handlers to be set up programatically instead of declaratively... I guess because the properties like `showTestMessage` aren't global anymore?